sstables/sstable_set.cc

```diff
@@ -769,6 +770,393 @@ sstable_set_impl::create_single_key_sstable_reader(
     return make_combined_reader(schema, std::move(permit), std::move(readers), fwd, fwd_mr);
 }
 
+static logging::logger cormlogger("clustering_order_reader_merger");
```

Comment:
We usually name the logger after the file they are in. We don't create a logger for each class.